sashimi-webapp/src/components/editor-viewer/Navbar.vue

```diff
@@ -64,10 +70,41 @@
 </script>
 
 <style scoped lang="scss"> 
-@import '../../assets/styles/variables.scss';
+@import 'src/assets/styles/variables.scss';
 
 .navbar {
-  border-bottom: 1px solid black;
+  box-sizing: border-box;
+  border-bottom: 3px solid #bebebe;
+  box-shadow: 0 2px 10px #e6e6e6;
```

Comment:
`box-shadow` should be used with a transparent color (`rgba`) so that the shadow will blend in with the background color.

None transparent color should only be used here as a compatibility fallback for older browser. e.g. 
```css
.navbar {
  box-shadow: 0 2px 10px #e6e6e6;
  box-shadow: 0 2px 10px rgba(0,0,0,0.3);
}
```